src/ontology/mondo-edit.obo

```diff
@@ -512847,6 +512847,9 @@ synonym: "DEE64" EXACT ABBREVIATION [DOID:0070375, OMIM:618004]
 synonym: "developmental and epileptic encephalopathy 64" EXACT [DOID:0070375, OMIM:618004]
 synonym: "EIEE64" EXACT ABBREVIATION []
 synonym: "epileptic encephalopathy, early infantile, 64" EXACT [OMIM:618004]
+synonym: "RHOBTB2 syndrome" EXACT []
```

Comment:
We really try to have a x-ref for all synonyms because it indicates that it is an actual name that is used by someone. 
In this case, maybe you can use the user's orcid. 
https://orcid.org/0000-0001-9310-0163